### PR TITLE
[QPPSF-2762] Updated PI Measures

### PR DESCRIPTION
**Pull request by NateTheGreatt**
Added and corrected formatting issues for new `pi-measures.json` file.

https://jira.cms.gov/browse/QPPSF-2762

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "mocha": "5.2.0",
     "pdf-to-text": "0.0.6",
     "request": "2.87.0",
-    "request-promise": "4.2.2",
+    "request-promise": "^4.2.2",
```
Comment (kalvinwang) [inline package.json]: I think for security reasons (don't want packages to suddenly import new code) we want to pin all of our packages to a specific version... is the ^ necessary?

File: util/measures/2018/pi-measure-relations.json [hunk 2]
```diff
@@ -1,42 +1,52 @@
...
+  "PI_PHCDRR_3_MULTI": { "reportingCategory": "registryBonus", "substitutes": []},
+  "PI_PHCDRR_1_MULTI": { "reportingCategory": "performanceBonus", "substitutes": []},
+  "PI_PHCDRR_4_MULTI": { "reportingCategory": "registryBonus", "substitutes": []},
+  "PI_PHCDRR_2_MULTI": { "reportingCategory": "registryBonus", "substitutes": []},
 
-  "PI_IACEHRT_1": {"reportingCategory": "cehrtBonus", "substitutes": []}
-}
+  "PI_TRANS_PHCDRR_1_MULTI": { "reportingCategory": "performanceBonus", "substitutes": []},
+  "PI_TRANS_PHCDRR_3": { "reportingCategory": "registryBonus", "substitutes": ["PI_PHCDRR_5"]},
+  "PI_TRANS_PHCDRR_3_MULTI": { "reportingCategory": "registryBonus", "substitutes": []},
```
Comment (rockykitamura) [inline util/measures/2018/pi-measure-relations.json]: For the new measures:
```
PI_PHCDRR_1_MULTI
PI_PHCDRR_2_MULTI
PI_PHCDRR_3_MULTI
PI_PHCDRR_4_MULTI
PI_PHCDRR_5_MULTI

PI_TRANS_PHCDRR_1_MULTI
PI_TRANS_PHCDRR_2_MULTI
PI_TRANS_PHCDRR_3_MULTI
```

I'm wondering if the same substitute relationships needs to exist for the similar named measure.  One example, does `PI_PHCDRR_5_MULTI` need to be considered a substitute of `PI_TRANS_PHCDRR_3_MULTI` in the same way `PI_PHCDRR_5` is considered a substitute of `PI_TRANS_PHCDRR_3`? @sarahnw @NateTheGreatt 

Same situation for `PI_TRANS_PHCDRR_2_MULTI`, are there substitutes similar to `PI_TRANS_PHCDRR_2`?

Comment (sarahnw) [reply util/measures/2018/pi-measure-relations.json]: good question. I don't totally know what the substitute field is for, if that's for scoring or policy. let me check with PI policy.

Comment (sarahnw) [reply util/measures/2018/pi-measure-relations.json]: @rockykitamura policy says yes they should

Comment (rockykitamura) [reply util/measures/2018/pi-measure-relations.json]: Great.  @NateTheGreatt will make the required updates.

Comment (NateTheGreatt) [reply util/measures/2018/pi-measure-relations.json]: @sarahnw @rockykitamura none of the aformentioned measureIds have those substitute values in the provided `pi-measures.json` file in the ticket https://jira.cms.gov/browse/QPPSF-2762

**Pull request merged**